fileonfeature.txt

```diff
@@ -1,2 +1,3 @@
 this is my file on feature branch
 added some lines to edit the file
+New line from new branch
```

Comment:
This is not the way how we add new lines to our txt files. It should start with 'added' for e.g.